my_internet_speed/__init__.py

```diff
@@ -31,6 +32,9 @@ def save(self):
         return Result.create(**data)
 
     def tweet(self):
+        if POST_TWITTER == False:
+            return
+        
         if self.result.download >= MINIMUM_SPEED:
```

Comment:
Other changes in this file can be cleaned up following my suggestion on `settings.py`. Beyond that we might have it all in one line:

```python
    if not TWEET or self.result.download >= MINIMUM_SPEED:
```